docs/containers-auth.json.5.md

```diff
@@ -24,7 +24,7 @@ is created by a `login` command from a container tool such as `podman login`,
 `buildah login` or `skopeo login`.  Each entry includes the name of the registry and then an auth
 token in the form of a base64 encoded string from the concatenation of the
 username, a colon, and the password. The registry name can additionally contain
-a path or repository name (an image name without tag or digest). The path (or
+a repository name (an image name without tag or digest) and namespaces. The path (or
```

Comment:
“entry includes the name of the registry [unrelated text, period]. Registry name can contain (things that are not a registry name)” is very convoluted.

namespaces are parents of repositories, and this wording seems to suggest the opposite or perhaps that their presence is orthogonal.

Reply:
Reworked the paragraph a bit.